css/fonts.css

```diff
@@ -0,0 +1,7 @@
+@font-face {
+	font-family: 'GothamPro';
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Final-Axion-cite/pullRequest?prid=5980774)